i18n/en/docusaurus-plugin-content-docs/current/guides/faq.mdx

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 7
+---
+
+# Miscellaneous
```

Comment:
This really feels like `Miscellaneous` that doesn't fit anywhere else. 

Maybe we should remove it of find a proper location later. But this is a content discussion and not something for this translation PR.

\\cc @VolkerK62